### ISSUE TITLE
Privacy. abbreviation should have aria-label to avoind pronoucing as a word

### ISSUE THREAD

**Issue by amazingphilippe** [cds-snc/notification-planning#111]
### Description of issue

In the link "Communications publiques (POU 914)." POU is read as a word, instead as individual letters.

### Potential fix

- We can wrap this abbreviation (and others) in the <abbr> tag and provide a title with the expansion of POU

### Resources
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/abbr

**Comment by amazingphilippe** [comment]
@sharlychan-cds I wonder what "PSU 914" ("POU 914" in French) stands for in this instance. [From what I can see on this page](https://www.canada.ca/en/treasury-board-secretariat/services/access-information-privacy/access-information/information-about-programs-information-holdings/standard-personal-information-banks.html#psu914), it isn't an acronym. any ideas what it is? And how it should be pronounced?
